apply.php

```diff
@@ -17,7 +17,7 @@
 	<div id="myCarousel" class="carousel slide" data-ride="carousel">
 		<div class="carousel-inner">
 			<div class="item active">
-				<img src="imgs/coffeeBanner.jpeg"" alt=" Los Angeles" style="width:100%;">
+				<img src="imgs/coffeeBanner.jpeg"" alt=" Los Angeles" height="500px" width="100%">
```

Comment:
![image](https://user-images.githubusercontent.com/888231/112075146-4ec25280-8b4e-11eb-8220-bcd113ae7d74.png)
 thats not better